examples/ewatercycle_api_notebook.ipynb

```diff
@@ -0,0 +1,496 @@
+{
```

Comment:
<p>Nice! Could we also create this structure while doing a more explicit loop in a nice way?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eWaterCycle/ewatercycle/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>